gc/base/standard/EnvironmentStandard.hpp

```diff
@@ -93,7 +95,9 @@ class MM_EnvironmentStandard : public MM_EnvironmentBase
 	 */
 	void flushRememberedSet()
```

Comment:
Should we wrap the function in a #if defined or just the code inside?  I am not sure about this one.  Rememberedset in this context is very scavenger specific so should anyone even call this unless they are using the scavenger?